### PR TITLE
Packages Initial Commit

### PR DESCRIPTION
**Pull request by ghost**
Desktop Fix, Mobile Fix, and Hover-Interactions For Desktop.

### CODE REVIEW

**Comment by ghost** [comment]
d2


whatsapp image 2018-06-15 at 6 39 45 pm

File: index.html [hunk 1]
```diff
@@ -12,7 +12,6 @@
     <link rel="stylesheet" type="text/css" href="styles.css" />
     <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
```
Comment (dsouzaedison) [inline index.html:13]: Make script tags inline element

Comment (ghost) [reply index.html:13]: They are in line. Plus script is not an element I think. The scripts are totally fine for now I think.

Comment (dsouzaedison) [reply index.html:13]: Just read about inline and block elements when you get time.

**Pull request merged**